students/Lincoln_Zhang/Session4/test_trigrams.py

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env python3
+import trigrams,os
+
+global_result = trigrams.read_file("sherlock_small.txt")
+
+def test_read_file():
```

Comment:
the goal is to have unit test be stand-alone -- so better to have the test(s) that test , e.g. file reading, actually read the file as part of the test, rather than as a global.